discovery-provider/plugins/notifications/src/__tests__/server/healthCheck.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect, jest, test } from '@jest/globals'
+import { renderEmail } from '../../email/notifications/renderEmail'
+import { Processor } from '../../main'
+import {
+  AppEmailNotification,
+  ReactionNotification
+} from '../../types/notifications'
+
+import {
+  createUsers,
+  insertMobileDevices,
+  insertMobileSettings,
+  dropTestDB,
+  createUserTip,
+  createReaction,
+  setupTest
+} from '../../utils/populateDB'
+
+import request from 'supertest'
+
+describe('Server Health Check Notifications', () => {
+  let processor: Processor
+  // Mock current date for test result consistency
+  Date.now = jest.fn(() => new Date('2020-05-13T12:33:37.000Z').getTime())
```

Comment:
for my learning where does this mock get used?

Reply:
For this test it actually doesnt. Will remove